src/main/views/search.njk

```diff
@@ -20,7 +20,7 @@
                     errorList: [
                         {
                             text: errorMessage,
-                            href: "#"
+                            href: "#search-input-container"
```

Comment:
the anchor tag here doesnt return focus to the input from when i tried it, not sure if youre using the right ref, also if you are using the right ref, can we rename? having `.../#search-input-container` in the url doesnt look fab

Reply:
It doesn't focus it jumps to that element on the page.